cli_research.py

```
@@ -0,0 +1,145 @@
+            "summarization_model": "deepseek:deepseek-chat",  # 总结用普通模型节省成本
+            "summarization_model_max_tokens": 4096,
+            "research_model": model,  # 使用用户指定的模型
+            "research_model_max_tokens": 8192,
+            "compression_model": "deepseek:deepseek-chat",  # 压缩用普通模型
+            "compression_model_max_tokens": 4096,
+            "final_report_model": model,  # 最终报告用用户指定的模型
+            "final_report_model_max_tokens": 8192,
+        }
+    }
+
+    # 如果提供了文档路径，添加MCP配置
```

Ruff (0.13.1)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)